parser/parser.go

```diff
@@ -45,12 +47,27 @@ type Test struct {
 	Time int // in milliseconds
 }
 
+// Benchmark contains the results of a single benchmark.
+type Benchmark struct {
+	Name     string
+	Duration time.Duration
+	// number of B/op
+	Bytes int
+	// number of allocs/op
+	Allocs int
+	// number of times this benchmark has been seen (for averaging).
+	Count  int
```

Comment:
Let's remove `Count` from the `Benchmark` struct (see my other comment below about simplifying the parser).

Reply:
I guess that's fine. I was trying to encompass all of the items we'd talked about on the issue, which includes the `-count` and `-benchmem` arguments.

If this works out, I can always add them back in a way that would make the most sense (maybe inside the system-out tag?).

Reply:
In one of my other comments I said that we should move the code that calculates the averages to the formatter instead. In that case the `Count` field would be unused, which is why I suggested to remove it here. It would still support multiple runs of a single benchmark via `-count`.

Reply:
Ah! Okay, thanks.